# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/media/webrtc/MediaEngineWebRTCAudio.h
# Commit: 3aef6c404e07
# Full Hash: 3aef6c404e07742214a5e3c1ccbcbd2ad1eee959
# Author: Chun-Min Chang <chun.m.chang@gmail.com>
# Date: 2022-01-28 04:41:16
# Regressor Bug: 1742655
# File Overlap Count: 2
# Description:
#   Bug 1742655 - Rename AudioInputTrack to AudioProcessingTrack r=pehrsons
#   
#   Depends on D134628
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D134629
# ==============================================================================

diff -r b004b579d1ce -r 3aef6c404e07 dom/media/webrtc/MediaEngineWebRTCAudio.h
--- a/dom/media/webrtc/MediaEngineWebRTCAudio.h	Thu Jan 27 22:29:35 2022 +0000
+++ b/dom/media/webrtc/MediaEngineWebRTCAudio.h	Thu Jan 27 22:29:35 2022 +0000
@@ -16,7 +16,7 @@
 namespace mozilla {
 
 class AudioInputProcessing;
-class AudioInputTrack;
+class AudioProcessingTrack;
 
 // This class is created and used exclusively on the Media Manager thread, with
 // exactly two exceptions:
@@ -92,9 +92,9 @@
   // The current preferences for the APM's various processing stages.
   MediaEnginePrefs mCurrentPrefs;
 
-  // The AudioInputTrack used to inteface with the MediaTrackGraph. Set in
+  // The AudioProcessingTrack used to inteface with the MediaTrackGraph. Set in
   // SetTrack as part of the initialization, and nulled in ::Deallocate.
-  RefPtr<AudioInputTrack> mTrack;
+  RefPtr<AudioProcessingTrack> mTrack;
 
   // See note at the top of this class.
   RefPtr<AudioInputProcessing> mInputProcessing;
@@ -201,7 +201,7 @@
   // operates in "pull" mode, and we append silence only, releasing the audio
   // input track.
   bool mEnabled;
-  // Whether or not we've ended and removed the AudioInputTrack.
+  // Whether or not we've ended and removed the AudioProcessingTrack.
   bool mEnded;
   // When processing is enabled, the number of packets received by this
   // instance, to implement periodic logging.
@@ -217,7 +217,7 @@
 };
 
 // MediaTrack subclass tailored for MediaEngineWebRTCMicrophoneSource.
-class AudioInputTrack : public ProcessedMediaTrack {
+class AudioProcessingTrack : public ProcessedMediaTrack {
   // Only accessed on the graph thread.
   RefPtr<AudioInputProcessing> mInputProcessing;
 
@@ -234,11 +234,11 @@
   // Only accessed on the main thread.
   Maybe<CubebUtils::AudioDeviceID> mDeviceId;
 
-  explicit AudioInputTrack(TrackRate aSampleRate)
+  explicit AudioProcessingTrack(TrackRate aSampleRate)
       : ProcessedMediaTrack(aSampleRate, MediaSegment::AUDIO,
                             new AudioSegment()) {}
 
-  ~AudioInputTrack() = default;
+  ~AudioProcessingTrack() = default;
 
  public:
   // Main Thread API
@@ -252,7 +252,7 @@
   Maybe<CubebUtils::AudioDeviceID> DeviceId() const;
   void Destroy() override;
   void SetInputProcessing(RefPtr<AudioInputProcessing> aInputProcessing);
-  static AudioInputTrack* Create(MediaTrackGraph* aGraph);
+  static AudioProcessingTrack* Create(MediaTrackGraph* aGraph);
 
   // Graph Thread API
   void DestroyImpl() override;
@@ -273,7 +273,7 @@
                         size_t aFrames, TrackRate aRate, uint32_t aChannels);
 
   // Any thread
-  AudioInputTrack* AsAudioInputTrack() override { return this; }
+  AudioProcessingTrack* AsAudioInputTrack() override { return this; }
 
  private:
   // Graph thread API
